src/cmake/compiler.cmake

```diff
@@ -655,7 +655,12 @@ if (CMAKE_SKIP_RPATH)
     unset (CMAKE_INSTALL_RPATH)
 else ()
     if (NOT CMAKE_INSTALL_RPATH)
-        set (CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_FULL_LIBDIR}")
+        if(APPLE)
+            set(BASEPOINT @loader_path)
```

Comment:
This is kind of reverting what was made before. You could pass `-DCMAKE_INSTALL_RPATH` on a per platform basis instead.

Reply:
Well, before, I think CMAKE_INSTALL_RPATH was defaulting to an absolute path; whereas now, it uses loader-path-relative defaults. We could still pass the option on a per-platform basis, but I'm not sure how that would be advantageous...?